postmates.py

```diff
@@ -0,0 +1,71 @@
+import json
+import datetime
+"""
+  monday: 10am -> 4pm
+  tuesday: 10am -> 4pm
+  wednesday: 11am -> 8pm
+  thursday: 11am -> 8pm
+  friday: 11am -> 2am
+  saturday: 5pm -> 4am
+"""
+# day of week: 1 -> Monday, 2 -> Tuesday, ..., 7 -> Sunday
+# time store as minutes since midnight
+input_json = '''{
+  "hours": [
+    {
+      "days": [1,2],
+      "hours": {"start_min": 600, "end_min": 960}
+    },
+    {
+      "days": [3,4],
+      "hours": {"start_min": 660, "end_min": 1200}
+    },
+    {
+      "days": [5],
+      "hours": {"start_min": 660, "end_min": 120}
+    },
+    {
+      "days": [6],
+      "hours": {"start_min": 1020, "end_min": 240}
+    }
+  ]
+}'''
+
+
+# {u'hours': {u'start_min': 600, u'end_min': 960}, u'days': [1, 2]}
+def is_store_open(example, dt):
```

Comment:
nice, this is more manageable, when you code, make sure to approach it as: "its not just for me"

now, lets improve this, imagine that this will be queried many times, how will this perform? and if its not good, what would do to improve it?